lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -69,6 +66,20 @@ def provision
       nil #void
     end
 
+    # Kill all instances.
+    #
+    # @param instances [Enumerable<EC2::Instance>]
+    # @return [void]
+    def kill_instances(instances)
+      instances.each do |instance|
+        if instance.exists?
```

Comment:
@anodelman Certainly.  (For what it's worth, if I recall correctly, I had done that because all of the callers guarantee that now themselves.)